src/commands/spawn.ts

```diff
@@ -7,10 +7,11 @@ import { blue, lightBlue } from '../formatting'
 const MAX_PORT = (2 ** 16) - 1
 
 export default class Spawn extends Command {
-  static description = 'Spawns a session backend from the provided docker image.'
+  static description = 'Spawns a session backend with the provided service\'s docker image.'
 
   static examples = [
     '<%= config.bin %> <%= command.id %> my-service',
+    '<%= config.bin %> <%= command.id %> my-service prod',
     '<%= config.bin %> <%= command.id %> my-service -e SOME_ENV_VAR=foo -e ANOTHER_ENV_VAR=bar',
```

Comment:
I'm a little worried about the combining of an optional argument and multiple arguments here, e.g. if someone does `my-service -e SOME_ENV_VAR=foo ANOTHER_ENV_VAR=bar` it will interpret `ANOTHER_ENV_VAR=bar` as the env name and fail with a potentially more confusing message than it would now.

I don't have a good solution though, I think the current way may be best

Reply:
What if the service arg allowed both `my-service` and `my-service/prod`?

Reply:
From offline convo, we're going to go with the approach I suggested.